build-scripts/verify_references.py

```diff
@@ -137,7 +137,8 @@ def is_remote_feed(href):
             href.startswith("security-oval-com.oracle") or \
             href.startswith("oval-com.ubuntu") or \
             href.startswith("pub-projects-security-oval-suse") or \
-            href.startswith('security-oval-oval-definitions-bookworm')
+            href.startswith('security-oval-oval-definitions-bookworm') or \
```

Comment:
```suggestion
            href.startswith('security-oval-oval-definitions-bookworm') or \
```
Is this needed?

Reply:
not sure what you mean there @Mab879 isn't your suggested change two identical lines?

i removed bullseye (as i accidentally left it in when i was testing) but bookworm needs to stay (with the trailing ` or \`)

Reply:
I'm sorry, I miss read the diff, that should be fine.